db/migrate/20230429100703_create_auto_tagger_bot.rb

```diff
@@ -0,0 +1,37 @@
+class CreateAutoTaggerBot < ActiveRecord::Migration[5.2]
+  def change
+    RequestStore.store[:skip_notifications] = true
+
+    meedan_team = Team.where(slug: 'meedan').last || Team.new(name: 'Meedan', slug: 'meedan')
+    meedan_team.skip_notifications = true
+    meedan_team.skip_clear_cache = true
+    meedan_team.skip_check_ability = true
+    meedan_team.save!
+
+    Team.current = meedan_team
+    tb = BotUser.new
```

Comment:
**thought:** I'm not sure how often we make botusers for the application, but given the amount of setup here it seems like it would be nice to add a constructor with any needed validations if we expect to do it again.